pkg/catalog/types.go

```diff
@@ -40,6 +40,7 @@ const (
 	// IndexTable has two column at most, the first is idx col, the second is origin table primary col
 	IndexTableIndexColName     = "__mo_index_idx_col"
 	IndexTablePrimaryColName   = "__mo_index_pri_col"
+	IndexTablePrimaryColNameSK = "__mo_index_pri_col_sk"
```

Comment:
Lets assume, this is the alias for OriginalTable's PK.

Reply:
Does this mean like 42 is not necessary any more?

Reply:
Is this comment true?   IndexTable has two columns at most.   Suppose the original table is 
create table t(pk1, pk2, sk1, sk2, primay key(pk1, pk2));
create index sk on t(sk1, sk2);

I suppose the 2nd index table should have 3 columns,  (sk1, ks2, __mo_index_pri_col_sk);
where __mo_index_pri_col_sk is the packed primary key (pk1, pk2).

Now seems you are saying we have (__mo_index_idx_col, __mo_index_pri_col_sk) instead.   Where 
__mo_index_Idx_col is the packed secondary key of (sk1, sk2).  

This is fine, all we really want at this stage is matching unique index.   Just I am somewhat confused.

Reply:
> Does this mean like 42 is not necessary any more?

42 is used by the IndexTable. It is the name of the 2nd column of IndexTable.

43 is going to be an alias for OriginalPK which will be inside the 1st column of IndexTable.

Reply:
> Is this comment true? IndexTable has two columns at most. Suppose the original table is

Yes. IndexTable has 2 columns

1st Column: "__mo_index_idx_col" --> serial(sk1, sk2, serial(pk1, pk2)) [Primary Key]
2nd Column:"__mo_index_pri_col"--> serial(pk1, pk2)